core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender42.java

```diff
@@ -37,13 +37,14 @@
  */
```

Comment:
I have a fundamental question. Since we are internally using Apache HTTP Client 4.5.3 to send the data, why do we need this sender in the first place? I think if there is anywhere still we are using HTTP Client 4.2 we should remove those. For eg `ApacheSenderFactory` looks for old jar on the class path and creates an instance of ApacheSender42. I think we do not need it at all. @littleaj can you take a look as a part of this improvements and trim the unnecessary code there ?

Reply:
That's something we can address separately. If it's worth exploring we should file an issue.

Reply:
Please file an issue to keep an eye on. We can then close if not needed.